### PR TITLE
fix: responsiveness of weather details card

### LINKED ISSUE #7: Improve mobile usability
The card that displays the weather information isn't responsive. It's far too narrow on mobile, and too wide on desktop. Can you look into making it be an appropriate size on all screen sizes.

### PR DESCRIPTION
**Pull request by CIPHERTron**
Fixes #7 
Collaborated with @psahithireddy to fix this issue 🚀 

---

https://user-images.githubusercontent.com/56754747/150573862-15811de2-80fe-4445-b427-58dc2fcd705b.mov

### CODE REVIEW

**Comment by CIPHERTron** [comment]
@ditsuke this PR is good for review 🚀

File: src/components/WeatherMap/WeatherMap.css [hunk 1]
```diff
@@ -4,3 +4,9 @@
   width: 80%;
   height: 300px;
 }
+
+@media (max-width: 844px) {
+  .leaflet-container {
+    width: 100%;
+  }
+}
```
Comment (ditsuke) [inline src/components/WeatherMap/WeatherMap.css:12]:
```suggestion
}

```

**Review by ditsuke — APPROVED** [review]
LGTM

**Review by harshsharma6401 — APPROVED** [review]

**Comment by ditsuke** [comment]
2 reviews and a successful deploy: merging this in!

**Pull request merged**